### PR TITLE
Add golang1.19

### PR DESCRIPTION
**Pull request by grisu48**
Add the container definitions for golang1.19

### CODE REVIEW

File: bci_tester/data.py [hunk 1]
```diff
@@ -188,6 +188,9 @@ def create_BCI(
 GO_1_18_CONTAINER = create_BCI(
     build_tag="bci/golang:1.18", image_type="hybrid"
 )
+GO_1_19_CONTAINER = create_BCI(
+    build_tag="bci/golang:1.19", image_type="hybrid"
```
Comment (dcermak) [inline bci_tester/data.py:192]: this container is only available for 15.4

**Review by dcermak — CHANGES_REQUESTED** [review]
The Go 1.19 container needs to be added to the metadata tests as well.

File: bci_tester/data.py (hunk 1 above)
Comment (grisu48) [reply bci_tester/data.py]: Updated, please check again.

**Review by dcermak — CHANGES_REQUESTED** [review]
Please add `GO_1_19_CONTAINER` to `CONTAINERS_WITH_ZYPPER` and adjust `test_metadata.py` as well.

**Comment by grisu48** [comment]
> Please add `GO_1_19_CONTAINER` to `CONTAINERS_WITH_ZYPPER` and adjust `test_metadata.py` as well.

Done.

**Comment by dcermak** [comment]
@grisu48 Can you please rebase on main?

**Comment by grisu48** [comment]
Rebased

**Comment by grisu48** [comment]
Thank you! :slightly_smiling_face:

**Pull request merged**